data/sc.json

```diff
@@ -0,0 +1,22 @@
+{
+    "state": "South Carolina",
+    "pages": [
+        {
+            "source": "",
+            "paragraphs": [
+               "\"To incorporate in the state of North Carolina you must follow the following steps:\r\n\r\nChoose your NC corporation filing option\r\nSubmit the NC Articles of Incorporation\r\nCreate North Carolina corporate bylaws\r\nGet a Federal EIN from the IRS\r\nOpen a corporate bank account\r\nObtain any required business licenses\r\nFile the North Carolina Annual Report\r\nPay taxes on the corporation\u2019s income\r\n1\"",
```

Comment:
a lot of these files seem to have their entire sentences quoted ie: 
```json
"\"To incorporate .... \""
```
vs
```json
"To incorporate ... "
```

This will cause the chat to quote everything in the response. Unless there's a grammatical reason for the quotes, they should be removed.